league_rpc_linux/lobby.py

```diff
@@ -0,0 +1,36 @@
+from league_rpc_linux.lcu.LCU import getLcuAPI
```

Comment:
Where is this located, can't see it in your PR.

Reply:
Oh, its legacy code from the first test - forgot to remove it after switching to lcu-driver. 
Going to remove it in a bit.